src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManager.groovy

```diff
@@ -36,6 +36,7 @@ class LSFJobManager extends AbstractLSFJobManager {
 
     final static String TIMESTAMP_PATTERN = /${MONTH}\s+${DAY}\s+${TIME}(\s+${YEAR})?(\s+${LSF_SUFFIX})?/
 
+    static final String LSF_COMMAND_ENVIRONMENT = "LSB_NTRIES=1"
```

Comment:
At least for Roddy, the number of retries can probably be fixed to one. For other circumstances the number may not be appropriate and may have to be configurable.

Consider making this a `LinkedHashMap` (ordered!). Otherwise, renaming the accessor to  `getEnvironmentString()` is also an option.